### PR TITLE
refactor: import in `Parser.scala`

### PR DESCRIPTION
**Pull request by ninaandrup**
#2512

### CODE REVIEW

File: main/src/ca/uwaterloo/flix/language/phase/Parser.scala [hunk 1]
```diff
@@ -721,9 +721,9 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
     def LetImport: Rule1[ParsedAst.Expression] = {
 
       // TODO: Remove me
-      def JvmStaticName: Rule1[Seq[String]] = rule {
+      /*def JvmStaticName: Rule1[Seq[String]] = rule {
```
Comment (magnus-madsen) [inline main/src/ca/uwaterloo/flix/language/phase/Parser.scala:724]: Just remove the old code :)

Comment (magnus-madsen) [reply main/src/ca/uwaterloo/flix/language/phase/Parser.scala:724]: Remember to check the diff on GitHub :)

Comment (ninaandrup) [reply main/src/ca/uwaterloo/flix/language/phase/Parser.scala:724]: Ah, yes, I will do that :-)

Comment (ninaandrup) [reply main/src/ca/uwaterloo/flix/language/phase/Parser.scala]: It should be corrected now

**Comment by magnus-madsen** [comment]
Neat!

**Pull request merged**